code/modules/clothing/masks/gasmask.dm

```diff
@@ -45,15 +54,73 @@ GLOBAL_LIST_INIT(clown_mask_options, list(
 	QDEL_LAZYLIST(gas_filters)
 	return..()
 
+/obj/item/clothing/mask/gas/equipped(mob/equipee, slot)
+	if(cig)
+		cig.equipped(equipee, slot)
+	return ..()
+
+/obj/item/clothing/mask/gas/adjustmask(mob/living/carbon/user)
+	if(cig)
+		user.visible_message(
+				span_warning("You need to remove [cig] before you can adjust the mask")
+		)
+		return
+	return ..()
+
+
 /obj/item/clothing/mask/gas/examine(mob/user)
 	. = ..()
-	if(max_filters > 0)
+	if(cig)
+		. += "<span class='notice'>There is a [cig.name] jammed into the filter slot</span>"
```

Comment:
```suggestion
		. += span_notice("There is a [cig.name] jammed into the filter slot.")
```

old pattern that never got fixed, how strange. This does the exact same thing. Also fixes punctuation.